src/scrape_up/google_maps/main.py

```diff
@@ -0,0 +1,162 @@
+"""This script serves as an example on how to use Python 
+   & Playwright to scrape/extract data from Google Maps"""
+   
+from playwright.sync_api import sync_playwright
+from dataclasses import dataclass, asdict, field
+import pandas as pd
+import argparse
+
+
+@dataclass
+class Business:
```

Comment:
Hey, in the code you submitted. It is not according to the project structure. 
You do not just have to submit a Python file, to perform the functionality. But in such a way that it can we used as a package. 

Check [this](https://github.com/Clueless-Community/scrape-up/blob/main/src/scrape_up/github/users.py) as a reference, how a class is initiated and methods are added to perform different functions.

Reply:
And do not use packages like `pandas`, etc. Go for `bs4`, `scrapy`, or `selenium`.

Reply:
Any update @okaditya84?

Reply:
> Any update @okaditya84?

I am Updating. Kindly give me a day or two. Thanks.